.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thanks for sending a pull request!
+
+If this is your first contribution, please review and sign our contributor agreement -
+https://www.elastic.co/contributor-agreement.
+
+See also https://github.com/elastic/apm-server/blob/master/CONTRIBUTING.md for more tips on contributing.
+-->
+
+<!--
+Replace this comment with a description of what's being changed by this PR.
+
+If this PR should close an issue, please add one of the magic keywords
+(e.g. fixes) followed by the issue number. For more info see:
+https://help.github.com/articles/closing-issues-using-keywords/
+
+Most changes should include:
+
+* unit tests
+* integration tests
+* documentation
+
+Major changes require a number of considerations including impact on:
+
+* logging selector(s)
+* metrics
+* telemetry
+* ESS/ECE
```

Comment:
Might not be clear to everyone what ESS/ECE is. Without linking to some docs (might not exist yet) this could be hard to follow.